doc/main.html

```diff
@@ -1235,6 +1249,272 @@ <h3>Git commit hooks</h3>
+      </p>
+      <p class="note">
+        See <a href="#bib-smpte-ag-07"></a> for more info regarding document process flowcharts and mapping these status points. 
+      </p>
+      <table id="tab-document-mapping-status" class="col-2-center col-3-center">
+        <caption>Mapping <code>pubState</code> and <code>pubStage</code> to Document Status</caption>
+        <thead>
+          <tr>
+            <th>Document Status</th>
+            <th><code>pubState</code> <code>content</code> value</th>
+            <th><code>pubStage</code> <code>content</code> value</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>Undergoing intial draft or revision in DG/SG/WG</td>
+            <td><code>WD</code></td>
+            <td><code>draft</code></td>
+          </tr>
+          <tr>
+            <td>Undergoing pre-FCD review (or) undergoing comment resolution from pre-FCD review (or) undergoing comment resolution from PCD period</td>
+            <td><code>CD</code></td>
+            <td><code>draft</code></td>
+          </tr>
+          <tr>
+            <td>Pre-FCD review complete (or) ready for FCD ballot (or) undergoing FCD ballot (or) ready for PCD (or) currently in PCD period</td>
+            <td><code>CD</code></td>
+            <td><code>pub</code></td>
+          </tr>
+          <tr>
+            <td>FCD ballot complete, undergoing comment resolution</td>
+            <td><code>FCD</code></td>
+            <td><code>draft</code></td>
+          </tr>
+          <tr>
+            <td>FCD ballot comment resolution complete (or) undergoing pre-DP review (or) ready for DP ballot (or) undergoing DP ballot </td>
+            <td><code>FCD</code></td>
+            <td><code>pub</code></td>
+          </tr>
+          <tr>
+            <td>FCD ballot complete with no comments (or) ready for ST Audit (or) undergoing ST audit</td>
+            <td><code>DP</code></td>
+            <td><code>pub</code></td>
+          </tr>
+          <tr>
+            <td>ST audit complete (or) waiting for publication</td>
+            <td><code>PUB</code></td>
+            <td><code>draft</code></td>
+          </tr>
+          <tr>
+            <td>Published</td>
+            <td><code>PUB</code></td>
+            <td><code>pub</code></td>
+          </tr>
+        </tbody>
+      </table>
+
+      <p>
+        See <a href="#sec-sample-workflow"></a> for a sample workflow utilizing these statuses. 
+      </p>
+
+    </section>
+
+    <section id="sec-document-github-releases">
+      <h3>GitHub Releases</h3>
+      <p>
+        Whenever the status of the document dictates as per <a href="#tab-document-mapping-status"></a>, that a merged commit to the main branch that updates the document to <code>&lt;meta itemprop="pubState" content="pub" /&gt;</code>, a release shall be created in the GitHub repo. 
```

Comment:
Whenever the state of the document changes to "pub" on the main branch, then a release SHALL be created.

Reply:
Addressed in https://github.com/SMPTE/html-pub/pull/139/commits/6ab6d4dd994fa0751b01ac792613709907ee3b96